guby_backend/views.py

```diff
@@ -77,6 +92,46 @@ def meetup_delete(request, meetup_id):
     return render(request, 'app/meetup_delete.html', {"form": form, "meetup_id": meetup.id})
 
 @login_required()
+@has_ownership
+def meetup_owner_index(request, meetup_id):
+   
+    if request.method == 'GET':
+        users = User.objects.filter(groups__name=f'meetup-owner#{meetup_id}')
+        # no group should become orphan by deleting all owners
+        if len(users) < 2:
```

Comment:
The `2` here will need rethinking at a later point. I think it makes more sense for us to be explicit like so:

```suggestion
        if len(users) == 1:
```

Thoughts?

Reply:
Also 1 more general thought here: At this time, once an owner is added to the group, they get all the same rights as the person who created the group. This means, I can add you and then you can kick me out. 

We can fix this later. Just making it explicit that this problem will exist.

Reply:
Yes, I see owners as users with equal rights as creators.
Which brings us to the point, should we remove `creator` field from the meetup object?